spec/features/newflow/educators/educator_signup_flow_spec.rb

```diff
@@ -0,0 +1,245 @@
+require 'rails_helper'
+
+module Newflow
+
+  feature 'Educator signup flow', js: true do
+
+    background { load 'db/seeds.rb' }
+    before(:each) { turn_on_educator_feature_flag }
+
+    let(:first_name) { "Elon#{Faker::Alphanumeric.alphanumeric(number: 20)}" }
+    let(:last_name) { "Musk#{Faker::Alphanumeric.alphanumeric(number: 20)}" }
+    let(:phone_number) { Faker::PhoneNumber.phone_number }
+    let(:email_value) { "#{Faker::Alphanumeric.alphanumeric(number: 20)}@rice.edu" }
```

Comment:
same here, faker can generate emails

Reply:
Faker generates emails, yes, but they are short words which could be a problem because SheerID requires user information to be unique, otherwise it gives a Verification Limit Exceeded error.

Reply:
There is a unique attribute you can add to emails (https://github.com/faker-ruby/faker/blob/master/doc/default/internet.md)
See: https://github.com/faker-ruby/faker#ensuring-unique-values

Reply:
Like I said, I'm okay with this - just sharing the realm of possibilities. :)

Reply:
Yeah, but that's only for every time that the tests run. And I did actually just add `unique` to `Faker::Alphanumeric.unique.alphanumeric(number: 30)` as an extra precaution.

Reply:
You know what? I changed my mind. I am not really submitting the info to the iframe in the tests (I just simulate it with `simulate_step_3_instant_verification`... so I am going to use the NAME generator. Good suggestion :).